cdc_admin/cdc_admin_handler.h

```diff
@@ -73,6 +74,12 @@ class CDCAdminHandler : virtual public CdcAdminSvIf {
   // of overriding some admin functions
   common::ObjectLock<std::string> db_admin_lock_;
 
+  virtual std::unique_ptr<replicator::DbWrapper> getDbWrapper(std::string db_name) {
+      std::unique_ptr<replicator::TestDBProxy> db_wrapper =
+          std::make_unique<replicator::TestDBProxy>(db_name, 0);
+      return db_wrapper;
+  }
+
```

Comment:
Instead of having a test implementation as default should we make this a pure virtual method (and make this an abstract class)?

Reply:
For the sake of the open source folks out there, I like having it work out of the box :) but I can change it if you disagree!

Reply:
I would prefer the default be the generic CDC implementation where we actually publish the key and value to Kafka. Maybe add a TODO for doing that in the future?

Reply:
I feel it will be hard to create a generic kafka publisher in the open source repo since others may configure their service and kafka clusters differently... My preference would be to leave it like this for now, and we can remember it as an internal TODO :)